src/python/dxpy/nextflow/ImageRef.py

```diff
@@ -133,7 +137,7 @@ def _reconstruct_image_ref(self):
         Docker image reference has the form of <REPOSITORY_NAME>/<IMAGE_NAME>:<VERSION_TAG> or
         <REPOSITORY_NAME>/<IMAGE_NAME>@<DIGEST>
         """
-        repo_and_image_name = self._join_if_exists("/", [self._repository, self._image_name])
+        repo_and_image_name = self._join_if_exists("", [self._repository, self._image_name])
```

Comment:
why the delimiter is removed here?

Reply:
Nextaur CLI outputs json with [repository](https://github.com/dnanexus/nextaur-app/blob/98a2e583286511f50ec9f35c116eef11a66516fb/plugins/nextaur/src/main/nextflow/cloud/dnanexus/fs/DxPathTokens.java#L33) with a trailing slash